common/js/src/popup-window/popup-opener.js

```diff
@@ -32,6 +32,24 @@ goog.require('goog.object');
 
 goog.scope(function() {
 
+const GSC = GoogleSmartCard;
+
+/**
+ * The window options to be used for the resulting window.
+ * APP mode supports all following options.
+ * EXTENSION mode supports 'width' only.
+ * @typedef {{
+ *            alwaysOnTop:(boolean|undefined),
+ *            frame:(string|undefined),
+ *            hidden:(boolean|undefined),
+ *            id:(string|undefined),
+ *            width:(number|undefined),
+ *            resizeable:(boolean|undefined),
+ *            visibleOnAllWorkspaces:(boolean|undefined)
+ *          }}
+ */
+let WindowOptions;
+
 const DEFAULT_DIALOG_CREATE_WINDOW_OPTIONS = {
```

Comment:
i looked at other examples using typedef and the format was like so, the typedef written and then the object declared write below so that it shows the object name of this type, are you suggesting something like so?
// WindowOptions type
let WindowOptions;

Reply:
I meant
```
/** @type {WindowOptions} */
const DEFAULT_DIALOG_CREATE_WINDOW_OPTIONS = {
```